src/components/Table/index.tsx

```diff
@@ -396,16 +396,24 @@ class Table<T> extends React.Component<TableProps<T>, TableState<T>> {
 		}
 	};
 
+	public setChange = (change: any) => {
```

Comment:
```suggestion
	public setPage = (change: any) => {
```

As setChange is two verbs that don't mean much :)

Reply:
done 👍